src/components/common/ColonyHome/ColonyFinishDeployment/ColonyFinishDeployment.tsx

```diff
@@ -18,31 +19,29 @@ import { getAllUserRoles } from '~modules/transformers';
 
 import styles from './ColonyFinishDeployment.css';
 
+const displayName = 'common.ColonyHome.ColonyFinishDeployment';
+
 const MSG = defineMessages({
   deploymentNotFinished: {
-    id: `dashboard.ColonyHome.ColonyFinishDeployment.deploymentNotFinished`,
+    id: `${displayName}.deploymentNotFinished`,
     defaultMessage: `Colony creation incomplete. Click to continue 👉`,
   },
   buttonFinishDeployment: {
-    id: `dashboard.ColonyHome.ColonyFinishDeployment.buttonFinishDeployment`,
+    id: `${displayName}.buttonFinishDeployment`,
     defaultMessage: `Finish Deployment`,
   },
 });
 
-type Props = {
-  colony: Colony;
-};
-
-const displayName = 'dashboard.ColonyHome.ColonyFinishDeployment';
-
-const ColonyFinishDeployment = ({
-  colony,
-  colony: { isDeploymentFinished, colonyAddress },
-}: Props) => {
+const ColonyFinishDeployment = () => {
   const { version: networkVersion } = useNetworkContracts();
   const { wallet } = useAppContext();
   const userHasAccountRegistered = useUserAccountRegistered();
 
+  const { colony } = useColonyContext();
+  const { colonyAddress } = colony || {};
+  /** @TODO: This should be a property of colony */
+  const isDeploymentFinished = false;
```

Comment:
It actually isn't since we won't use it anymore. 

(it will be removed along with the whole "check if deployment finished" logic)